scripts/build.py

```diff
@@ -152,12 +175,19 @@ def get_cache_slug():
         BUILD_DIR, 'third_party', 'generated')
     build_minified_third_party_libs(THIRD_PARTY_GENERATED_OUT_DIR)
 
-    # Process extensions, copy it to build/[cache_slug]/extensions
+    # Minify extension static resources, copy it to
+    # build/[cache_slug]/extensions
     EXTENSIONS_SRC_DIR = os.path.join('extensions', '')
     EXTENSIONS_OUT_DIR = os.path.join(BUILD_DIR, 'extensions', '')
-    copy_files_source_to_target(EXTENSIONS_SRC_DIR, EXTENSIONS_OUT_DIR)
-
-    _build_files()
+    # Certain files' syntax become incorrect after minification and hence
```

Comment:
Wait a sec. So what happens to the LogicProof files? Do they not get served in prod?

/cc @jacobdavis11